cddl/protected-corim-header-map.cddl

```diff
@@ -1,6 +1,16 @@
-protected-corim-header-map = {
-  &(alg: 1) => int
-  &(content-type: 3) => "application/rim+cbor"
-  &(corim-meta: 8) => bstr .cbor corim-meta-map
-  * cose-label => cose-value
-}
+; protected header map needs to contain at least one of corim-meta (8) or CWT-Claims (15)
+protected-corim-header-map =
+  {
+    &(alg: 1) => int,
+    &(content-type: 3) => "application/rim+cbor",
+    meta,
+    * cose-label => cose-value
+  }
+
+meta = ((
```

Comment:
following the convention in the rest of the CDDL this should be called "meta-group".